src/markdown/re/block.js

```diff
@@ -69,11 +88,8 @@ block.paragraph = replace(block.paragraph)
     ('comment', block.comment)
     ('table', table.normal)
     ('tablenp', table.nptable)
+    ('fences', block.fences.source.replace('\\1', '\\2'))
+    ('ol', block.list.block_ol.source.replace('\\1', '\\3'))
     ();
 
-block.paragraph = replace(block.paragraph)('(?!', '(?!'
```

Comment:
This replace was messing the whole thing up.

Reply:
I just grouped it as a classic replace, with the others.
The rest of the diff is prettying code.